### PR TITLE
Feature/2376 migrate reporting officer from dev v2

### PR DESCRIPTION
**Pull request by KHORCASH**
[AB#2376](https://dev.azure.com/dsd-sdsd/ad8a8cd2-ef49-4e7c-bae2-6f8729a38334/_workitems/edit/2376) Migrate reporting officer changes from dev-v2

### CODE REVIEW

**Review by jacobkellas — CHANGES_REQUESTED** [review]

File: Functions/RIPA.Functions.UserProfile/RIPA.Functions.UserProfile.csproj [hunk 1]
```diff
@@ -4,6 +4,11 @@
     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
     <UserSecretsId>81bf2225-559e-46d9-af96-af8b4d216835</UserSecretsId>
   </PropertyGroup>
+  <ItemGroup>
```
Comment (jacobkellas) [inline Functions/RIPA.Functions.UserProfile/RIPA.Functions.UserProfile.csproj:7]: I'm not sure how or why this showed up, but you can remove it.

File: Functions/RIPA.Functions.UserProfile/RIPA.Functions.UserProfile.csproj [hunk 2]
```diff
@@ -28,7 +33,4 @@
       <CopyToPublishDirectory>Never</CopyToPublishDirectory>
     </None>
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Functions\v2\" />
```
Comment (jacobkellas) [inline Functions/RIPA.Functions.UserProfile/RIPA.Functions.UserProfile.csproj:32]: Probably don't want to remove this?  I don't think?

File: UI/src/components/features/RipaPageContainer.vue [hunk 3]
```diff
@@ -32,6 +32,14 @@
       @on-save="handleSaveUser"
     ></ripa-user-dialog>
 
+    <ripa-snackbar
+      :text="officerGenderRaceText"
+      :auto-close="true"
```
Comment (jacobkellas) [inline UI/src/components/features/RipaPageContainer.vue:37]: This should be auto-close="false" so the use has to manually close it.

**Review by jacobkellas — CHANGES_REQUESTED** [review]

File: Functions/RIPA.Functions.UserProfile/RIPA.Functions.UserProfile.csproj (hunk 1 above)
Comment (KHORCASH) [reply Functions/RIPA.Functions.UserProfile/RIPA.Functions.UserProfile.csproj:7]: I'm getting weird behavior when I remove this ItemGroup, could we review this item?

Comment (jacobkellas) [reply Functions/RIPA.Functions.UserProfile/RIPA.Functions.UserProfile.csproj:7]: Sure thing, I'll pull it locally and take a look as well.

**Pull request merged**